pkg/grpc/client.go

```diff
@@ -0,0 +1,35 @@
+// Package grpc provides the basic interfaces to build a gRPC+Protobuf flows client & server
+package grpc
+
+import (
+	"github.com/netobserv/netobserv-agent/pkg/pbflow"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// ClientConnection wraps a gRPC+protobuf connection
+type ClientConnection struct {
+	client pbflow.CollectorClient
+	conn   *grpc.ClientConn
+}
+
+func ConnectClient(address string) (*ClientConnection, error) {
+	// TODO: allow configuring some options (keepalive, backoff...)
```

Comment:
Maybe set a default Backoff / Timeout in the meantime, before implementing it as a  configuration ?

Reply:
There are actually default values:
```go
var Default[Config](https://pkg.go.dev/google.golang.org/grpc/backoff#Config) = Config{
	BaseDelay:  1.0 * [time](https://pkg.go.dev/time).[Second](https://pkg.go.dev/time#Second),
	Multiplier: 1.6,
	Jitter:     0.2,
	MaxDelay:   120 * [time](https://pkg.go.dev/time).[Second](https://pkg.go.dev/time#Second),
}
```

Reply:
ah, nice! I remember having to debug tricky bugs in the past due to not having a timeout, but that wasn't grpc, glad this one has a default :)